src/sdk/accountspace_fs.js

```diff
@@ -46,15 +40,18 @@ class AccountSpaceFS {
      */
     constructor({ config_root, fs_root, config_root_backend, stats }) {
         this.config_root = config_root;
-        this.accounts_dir = path.join(config_root, CONFIG_SUBDIRS.ACCOUNTS);
-        this.access_keys_dir = path.join(config_root, CONFIG_SUBDIRS.ACCESS_KEYS);
-        this.buckets_dir = path.join(config_root, CONFIG_SUBDIRS.BUCKETS);
-        this.fs_context = native_fs_utils.get_process_fs_context();
+        this.config_root_backend = config_root_backend ?? config.NSFS_NC_CONFIG_DIR_BACKEND;
+        this.stats = stats;
+        this.fs_context = native_fs_utils.get_process_fs_context(
+            this.config_root_backend,
+            config.NSFS_WARN_THRESHOLD_MS,
+            this.stats?.update_fs_stats
+        );
+
+        this.configFS = new ConfigFS(config_root, this.config_root_backend, this.fs_context);
 
         // Currently we do not use these properties
         this.fs_root = fs_root ?? '';
```

Comment:
It was copied from bucketspace_fs, I see it empty also there, maybe we can remove it in both places?

Reply:
this is unrelated to this change, let's be focused on reviewing the relevant changes
I agree we can and need to remove it, open a github issue pls.